src/App.vue

```diff
@@ -1,38 +1,136 @@
 <script setup lang="ts">
+
   import Header from './components/Header.vue'
+
+  import CardSingle from './components/CardSingle.vue'
+  import CardGroup from './components/CardGroup.vue'
+  import CardSingleSource from './components/CardSingleSource.vue'
+  import CardSingleSocial from './components/CardSingleSocial.vue'
+
   import TopStory from './components/TopStory.vue'
   import MultipleHeadlines from './components/MultipleHeadlines.vue'
   import SingleVideo from './components/SingleVideo.vue'
   import SingleSourceRow from './components/SingleSourceRow.vue'
   import MayHaveMissed from './components/MayHaveMissed.vue'
   import Mastodon from './components/Mastodon.vue'
+
   import { getRandomNumbersInRange } from './components/helpers/sprinkle_getRandomNumbersInRange'
   
   var row1: number = getRandomNumbersInRange(1,0,24)
   var row2: number = getRandomNumbersInRange(1,0,24)
   var row3: number = getRandomNumbersInRange(1,0,24)
   var row4: number = getRandomNumbersInRange(1,0,24)
+
 </script>
 
 <template>
   <Header msg="scoopy" />
   <section class="module-row">
 
-    <TopStory headline="" body="" />
-    <MultipleHeadlines :count=4 />
-    <MultipleHeadlines :count=4 />
-    <MayHaveMissed />
-    <Mastodon :count=3 />
-    <SingleSourceRow :source=0 :count=50 />
-    <SingleVideo />
-    <SingleVideo />
-    <SingleSourceRow :source=row1 :count=50 />
-    <SingleSourceRow :source=row2 :count=50 />
-    <MultipleHeadlines :count=4 />
-    <MayHaveMissed />
-    <MultipleHeadlines :count=4 />
-    <SingleSourceRow :source=row3 :count=50 />
-    <SingleSourceRow :source=row4 :count=50 />
+    <CardSingle
+      card_title="Top Story"
+      card_description="A single card. Only has one bit of content (article) inside, and adds a card footer."
+      >
+      <TopStory />
+    </CardSingle>
+
+    <CardGroup
+      card_title="Just the Headlines"
```

Comment:
i dont like any of this actually passing as arguments. should be decided by the card type or choc sauce. i know you didnt add them but im just calling it out

Reply:
I did actually add them. I like the idea of somehow customizing them, at least while we're building it out. I vote we leave it until a replacement is made.

Reply:
yeah fine to keep until we have a replacement